ingestion-beam/src/main/java/com/mozilla/telemetry/util/Json.java

```diff
@@ -140,6 +165,11 @@ public static PubsubMessage readPubsubMessage(String data) throws IOException {
     return output;
   }
 
+  /** Convert a tree to a map of strings to strings, as appropriate for attributes. */
+  public static <T> T convertValue(ObjectNode root, Class<T> klass) {
+    return MAPPER.convertValue(root, klass);
```

Comment:
`MAPPER::convertValue` doesn't take advantage of the type of `root`, but `MAPPER::treeToValue` does:
```suggestion
    return MAPPER.treeToValue(root, klass);
```